worker/src/RTC/TransportCongestionControlServer.cpp

```diff
@@ -262,12 +282,26 @@ namespace RTC
 
 		// Increment packet count.
 		this->transportCcFeedbackPacket->SetFeedbackPacketCount(++this->transportCcFeedbackPacketCount);
+		this->transportCcFeedbackStartSeqNum = latestWideSeqNumber;
+	}
+
+	inline void TransportCongestionControlServer::MayDropOldPacketArrivalTimes(
+	  uint16_t seqNum, uint64_t nowMs)
+	{
+		MS_TRACE();
 
-		// Pass the latest packet info (if any) as pre base for the new feedback packet.
-		if (latestTimestamp > 0u)
+		if (nowMs >= PacketArrivalTimestampWindow)
```

Comment:
How can the current time (`nowMs`) be compared with a constant value?

Reply:
Because `nowMs` is a relative time, it's to prevent `nowMs` < 500 and `expiryTimestamp` will be negetive. but normally it'll not happened

Reply:
Because `nowMs` is a relative time, it's to prevent `nowMs` < 500 and `expiryTimestamp` will be negetive. but normally it'll not happened

Reply:
What do you mean with "relative time"? `nowMs` is indeed relative to a randomly chosen system value (it's not "current absolute value") but that's not a problem since we always compare `nowMs` with a previously obtained `nowMs`. However here you are comparing `nowMs` with a `500` constant.

Do you mean that this `nowMs` is not same as any other `nowMs` in the whole code and it's a diff instead? Otherwise I don't understand.

Reply:
Please @penguinol, write the corresponding comment (ie: Ignore nowMs value is it's smaller than 500 in order to avoid negativa values (Should never happen)) and return early if the condition is met.

```c++
if (nowMs < PacketArrivalTimestampWindow)
{
  return;
}
```

Reply:
Please @penguinol, do an early return of the method if the condition is not met, for readability.

Reply:
Yes, please:

```c++
if (nowMs < PacketArrivalTimestampWindow)
{
  return;
}

// Rest of the code.
```